rejson/client.py

```diff
@@ -68,8 +73,47 @@ def __init__(self, encoder=None, decoder=None, *args, **kwargs):
                 'JSON.OBJLEN': long,
         }
         for k, v in six.iteritems(MODULE_CALLBACKS):
-            self.set_response_callback(k, v)
-                                    
+            self.client.set_response_callback(k, v)
+
+    #######################################################################
+    # TODO - see if this pass-through stuff can be simplified
+    @property
+    def client(self):
+        return self._client
+
+    def execute_command(self, *args, **kwargs):
```

Comment:
@radoye why not creating a decorator for all the client functions?

Reply:
Sure. I'm happy to clean up once it's ready for merging. 

1. Let's figure out if tests make sense, or I overlooked something.
2. Except for hiding these trivial functions with a decorator, any other approach comments?